.buildkite/pipeline.yml

```diff
@@ -0,0 +1,8 @@
+steps:
+  - label: ':docker: Push Image'
+    command: ./ci-scripts/push-image.sh
+    env:
+      BINARY_OS: linux
+      BINARY_ARCH: amd64
+      IMAGE_REPOSITORY: 278521702583.dkr.ecr.us-west-2.amazonaws.com/iops/terraform-provider-mysql
+      IMAGE_TAG: "build-${BUILDKITE_BUILD_NUMBER}"
```

Comment:
Will tagged releases get a proper version tag?

Reply:
Nevermind - read the code below

Reply:
yes...just like apps

Reply:
logic is in the push-image.sh